Assets/Scripts/EOSSessionsManager.cs

```diff
@@ -574,13 +632,31 @@ public void OnLoggedOut()
             userLoggedIn = false;
         }
 
+        /// <summary>
+        /// Unsubscribes from invitations and Peer2Peer message requests.
+        /// This should be called before the application finishes shutting down in order to properly remove Native notifications.
+        /// TODO: This function has no references, and is incomplete in its implementation. This will likely be removed.
+        /// </summary>
+        [Obsolete]
         private void OnShutDown()
         {
             DestroyAllSessions();
             UnsubscribeFromGameInvites();
             UnsubscribeToSessionMessageConnectionRequests();
         }
 
+        /// <summary>
+        /// Informs the <see cref="UIInterface"/> that a UI Event has been processed.
+        /// Some functions in the EOS SDK return a UIEventId. When this happens, it is required that this function be called to released by the SDK.
+        /// For example, if you are subscribed to <see cref="OnJoinSessionAcceptedListener(ref JoinSessionAcceptedCallbackInfo)"/>, after a successful call to <see cref="SessionsInterface.JoinSession(ref JoinSessionOptions, object, OnJoinSessionCallback)"/>,
+        /// the notification function will receive a <see cref="JoinSessionAcceptedCallbackInfo.UiEventId"/>.
+        /// You can use this event id to create a <see cref="SessionDetails"/> object by calling <see cref="MakeSessionHandleByEventId(ulong)"/>.
+        /// After you've done so, or if you aren't going to use this handle, you must call this function to release the handle in the EOS SDK.
```

Comment:
An example of how to rephrase to remove pronouns:

```csharp
/// For example, if a subscription is made to <see cref="OnJoinSessionAcceptedListener(ref JoinSessionAcceptedCallbackInfo)"/>, after a successful call to <see cref="SessionsInterface.JoinSession(ref JoinSessionOptions, object, onJoinSessionCallback)"/>, 
/// the notification function will receive a <see cref="JoinSessionAcceptedCallbackInfo.UiEventId"/>.
/// This event id can be used to create a <see cref="SessionDetails"/> object by calling <see cref="MakeSessionHandleByEventId(ulong)"/>.
/// After doing so, this function must be called to release the handle within the EOS SDK.
```

Reply:
Seems like a clear rewrite, I'll use most of your text. I do think the "or if you aren't going to use this handle" is important, so I preserved it in the form of "or if the handle will not be utilized". Otherwise I think it implies "after using this call Acknowledge" rather than what is really intended of "after it's not useful then call Acknowledge".